Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -921,19 +922,19 @@ void GetBPRegInfo(const u8* data, std::string* name, std::string* desc)
     const char* logicmodes[] = {"0",     "s & d",  "s & ~d",   "s",        "~s & d", "d",
```

Comment:
Just wondering, would this make any difference being `constexpr` (and/or `std::array`, or anything other than `const char*[]`)?

Reply:
There would likely not be much of a difference here.